src/app/component/user/components/profile/edit-profile/social-networks/social-networks.component.scss

```diff
@@ -77,5 +80,15 @@ ul {
         }
       }
     }
+
+    li:last-child {
+      .link {
+        button {
+          img {
```

Comment:
I'm not sure, but in my opinion, the selector doesn't look good
why don't you just add a class to img?

Reply:
Added classes to all elements, so the selectors are simplier